<?xml version="1.0" encoding="utf-8"?>
<run passed="0" failed="1" unexecuted="0" time="21168" total="1">
  <tests>
    <test key=".\SalaryCostApi" name="SalaryCostApi" outcome="failed" time="20541">
      <step id="5erTiX6FiI0ZwnWyljT8Sl" name="CreateId" outcome="passed" time="51">
        <action id="2J6tsbc8viWtqwq8MTntZ6" type="coded-step" outcome="passed" time="42" />
      </step>
      <step id="7jBdhd1HWHjbBwo9mV3KZc" name="Http Request" outcome="failed" time="20434">
        <action id="5Qpa4pp1E5DUSPwA4hEEnI" type="http-request" outcome="failed" time="20434">
          <error message="The operation has timed-out." />
        </action>
      </step>
    </test>
  </tests>
  <output>
    <output message="The test case is starting ..." />
    <output message="Create Excel Result File" />
    <output message="Time of starting Test :: _05012023_11_22_13" />
    <output message="File Not Find" />
    <output message="26424" />
    <output message="Print environment arc" />
    <output message="Executing HTTP Request: PUT https://arc.salarycostsapi.is.ssg/api/v1/SalaryCosts" />
    <output message="The test case is starting ..." />
    <output message="" />
    <output message="Ending the test case ..." />
  </output>
  <errors>
    <error message="Running test's 'OnAfterTestCompleted' method failed. Message: 'Index was outside the bounds of the array.'" />
  </errors>
</run>